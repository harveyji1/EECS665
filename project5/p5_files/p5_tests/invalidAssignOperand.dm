f: () void{
    a: int;
    b: int;
    a = 1;
    a = b;
}

main: () void{
    f = 1;
}