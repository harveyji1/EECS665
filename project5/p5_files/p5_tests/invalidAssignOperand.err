FATAL [9,5]-[9,10]: Invalid assignment operation
Type Analysis Failed
