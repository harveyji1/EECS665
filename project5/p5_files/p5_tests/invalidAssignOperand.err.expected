FATAL [9,5]-[9,6]: Invalid assignment operand
Type Analysis Failed